core/src/main/java/org/apache/accumulo/core/util/LocalityGroupUtil.java

```diff
@@ -50,16 +52,16 @@
 
   // private static final Logger log = Logger.getLogger(ColumnFamilySet.class);
 
-  public static final Set<ByteSequence> EMPTY_CF_SET = Collections.emptySet();
+  public static final Set<ByteSequence> EMPTY_CF_SET = ImmutableSet.of();
```

Comment:
To take advantage of the ImmutableSet comparison logic, I needed this set to also be an ImmutableSet.